ecosystem/node-checker/src/evaluators/direct/api/state_sync.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use super::super::DirectEvaluatorInput;
+use super::ApiEvaluatorError;
+use crate::{
+    configuration::EvaluatorArgs,
+    evaluator::{EvaluationResult, Evaluator},
+    evaluators::EvaluatorType,
+};
+use anyhow::Result;
+use clap::Parser;
+use poem_openapi::Object as PoemObject;
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
+
+#[derive(Clone, Debug, Deserialize, Parser, PoemObject, Serialize)]
+pub struct StateSyncVersionEvaluatorArgs {
+    #[clap(long, default_value_t = 10000)]
```

Comment:
I think this might be too small. we generally run 500-1000 TPS, and might run up to 5k TPS. we should probably allow for a minute of delay, so 200000 might be a good number to use